physx-sys/src/lib.rs

```diff
@@ -241,8 +241,19 @@ pub struct FilterShaderCallbackInfo {
     pub constantBlockSize: u32,
 }
 
+pub type PxAgain = bool;
```

Comment:
I think just using a `bool` is fine.  The traits that expose the functions that return this just return a `bool`, so the end-user currently isn't seeing this at all.